src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -180,10 +181,26 @@ const convertFrameworkSummary = (framework) => {
 const convertTaskDetail = async (taskStatus, ports, logPathPrefix) => {
   // get container ports
   const containerPorts = {};
-  if (ports) {
+  const hashFunc = (str) => {
+    const hexStr = crypto.createHash('md5').update(str).digest('hex');
+    return parseInt(hexStr.substring(0, 12), 16) +
+      parseInt(hexStr.substring(12, 24), 16) + parseInt(hexStr.substring(24, 32), 16);
```

Comment:
>, 32) [](start = 75, length = 5)

align with runtime, do not limit the string end.

Reply:
Fine. Although md5 is used fixed 128 bit, I think it's same